manual_flagging/yaml/inputs.yaml

```diff
@@ -12,9 +12,13 @@ manual_update: false
 
 # The values in housing_market_type, run_tri, and tri_stat_groups will tell the
 # sales val script which properties will be updated. The housing_market_type
-# list supports 'res_single_family', 'res_multi_family', and/or 'condos'.
-# The run_tri van be 1, 2, and/or 3, and the values for the tri_stat_groups
-# can be "current" or "og_mansueto".
+# list supports 'res_single_family', 'res_multi_family', 'res_all,' and/or
+# 'condos'. If we have a methodology that flags all of res, we will use
+# 'res_all'. If we need to have a separate methodology for single and
+# multi-family, we can use  those respective values. This means that if res_all
+# is selected, we can't use either of the other res market values, as it would
+# be redundant. The run_tri van be 1, 2, and/or 3, and the values for the
+# tri_stat_groups can be "current" or "og_mansueto".
```

Comment:
This change is implemented in the potential case where we want to flag all of res with the same methodology.

Reply:
Scrapped this change as I'm not sure it will be needed.